### DISCUSSION TITLE
Not able to transcribe Polish audio while English audio is easily transcribed; encoding issues?

### DISCUSSION THREAD

**Discussion by agatatroost** [openai/whisper#2179] [Q&A] [answered]
Hi all, I am trying to transcribe audio files in Polish to text using the Whisper Open AI. I am encountering an error which seems to be about encoding the text (possibly because of Polish letters such as ś, ż etc.)? When I run the same command to transcribe audio in English, the programme transcribes it with no issues. I've read the "help" of Whisper but there is no information on encoding (that I can find). Any tips to fix this?

The command:
`whisper audio.wav --language Polish > transcriptPL.txt ` 

(I also tried it without setting language to Polish; for English, just "whisper audioEnglish.wav > transcriptEN.txt" works well)

The error code:

```

C:\Users\agata\Desktop\Transcripts WP4> whisper audio.wav > transcriptPL.txt
C:\Python312\Lib\site-packages\whisper\transcribe.py:115: UserWarning: FP16 is not supported on CPU; using FP32 instead
  warnings.warn("FP16 is not supported on CPU; using FP32 instead")
Traceback (most recent call last):
  File "C:\Python312\Lib\site-packages\whisper\transcribe.py", line 478, in cli
    result = transcribe(model, audio_path, temperature=temperature, **args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python312\Lib\site-packages\whisper\transcribe.py", line 349, in transcribe
    print(make_safe(line))
  File "C:\Python312\Lib\encodings\cp1252.py", line 19, in encode
    return codecs.charmap_encode(input,self.errors,encoding_table)[0]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
UnicodeEncodeError: 'charmap' codec can't encode characters in position 110-111: character maps to <undefined>

C:\Users\agata\Desktop\Transcripts WP4>
```

**Comment by glangford** [comment, marked as answer]
Run whisper without using '>' for redirecting console output, for example 
`whisper audio.wav --language Polish --output_format txt`

FYI below (I am not a Windows user but I suspect this is the problem):

**Why utf8 characters get broken when piped in Windows platform?**
https://www.reddit.com/r/rust/comments/we4y3p/why_utf8_characters_get_broken_when_piped_in/

**Reply by agatatroost** [reply]
This works! Thank you
